docs/spec/staking/spec-technical.md

```diff
@@ -113,7 +129,7 @@ Candidate parameters are described:
  - RedelegatingShares: The portion of `IssuedDelegatorShares` which are
    currently re-delegating to a new validator
  - VotingPower: Proportional to the amount of bonded tokens which the validator
-   has if the candidate is a validator.
+   has if `Candidate.Status` is `Bonded`; otherwise it is equal to `0`
```

Comment:
Maybe confused you with of my own errors here.... Anyways the status should never be equal to 0, should always be set to one of the status options (defined in the code, check it out)